nautilus-extension/src/menu_provider.rs

```diff
@@ -211,7 +211,8 @@ fn connect_activate_signal(raw_menuitem: *mut NautilusMenuItem, activate_fn: uns
             Some(mem::transmute(activate_fn as *mut c_void)),
             data as *mut c_void,
             None,
-            GConnectFlags::empty()
+            G_CONNECT_AFTER
```

Comment:
Thanks for the pull request!

What is `G_CONNECT_AFTER`? Old commented code line can be removed.

Reply:
https://developer.gnome.org/gobject/stable/gobject-Signals.html#GConnectFlags

Seems like `GConnectFlags::empty()` actually just returns 0. So line 214 should really just be `0`.